Sources/SentrySwiftUI/Preview/PreviewRedactOptions.swift

```diff
@@ -1,14 +1,66 @@
 #if canImport(SwiftUI) && canImport(UIKit) && os(iOS) || os(tvOS)
 import Sentry
 
+/**
+ * Redaction options for the session replay masking preview.
+ *
+ * - Note: See ``SentryReplayOptions.DefaultValues`` for the default values of each parameter.
+ */
 public class PreviewRedactOptions: SentryRedactOptions {
+    /**
+     * Flag to redact all text in the app by drawing a black rectangle over it.
+     *
+     * - Note: See ``SentryReplayOptions.DefaultValues.maskAllText`` for the default value.
+     */
     public let maskAllText: Bool
+
+    /**
+     * Flag to redact all non-bundled image in the app by drawing a black rectangle over it.
```

Comment:
`l`: A typo, I believel
```suggestion
     * Flag to redact all non-bundled images in the app by drawing a black rectangle over them.
```